app-modules/report/src/Filament/Widgets/StatsOverviewReportWidget.php

```diff
@@ -43,7 +43,7 @@
 abstract class StatsOverviewReportWidget extends StatsOverviewWidget
 {
     #[Locked]
-    public string $cacheTag;
+    public string $cacheTag = 'default';
```

Comment:
This should not be given a default value. It needs to be set by a class that extends this.